### PR TITLE
#10347: Update documentation and sweep config for rsub and rdiv

### PR DESCRIPTION
**Pull request by mcw-anasuya**
### Ticket
#10347  

### Problem description
Documentation incomplete for rsub and rdiv ops with supported params

### What's changed
Updated doc and corresponding sweep test files
Added golden function

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11217884233) - PASSED
- [x] Run Sweeps

### CODE REVIEW

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 1]
```diff
@@ -422,7 +424,7 @@ void bind_unary_operation_with_dim_parameter(
 }
 
 template <typename unary_operation_t>
-void bind_unary_rdiv(py::module& module, const unary_operation_t& operation, const std::string& parameter_name_a, const std::string& parameter_a_doc, const std::string& parameter_name_b, const std::string& parameter_b_doc, const std::string parameter_b_value, const std::string& description) {
+void bind_unary_rdiv(py::module& module, const unary_operation_t& operation, const std::string& parameter_name_a, const std::string& parameter_a_doc, const std::string& parameter_name_b, const std::string& parameter_b_doc, const std::string parameter_b_value, const std::string& description, const std::string& note) {
```
Comment (Aswinmcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:427]: Please add note=" "

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 2]
```diff
@@ -1410,7 +1416,7 @@ void py_module(py::module& module) {
     detail::bind_unary_operation_with_fast_and_approximate_mode(module, ttnn::rsqrt);
 
     // Unaries with float parameter
-    detail::bind_unary_operation_with_float_parameter(module, ttnn::elu, "alpha", "The alpha parameter for the ELU function",
+    detail::bind_unary_operation_with_float_parameter(module, ttnn::elu, "alpha", "The alpha parameter for the ELU function", "",
```
Comment (Aswinmcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1419]: And remove here

**Review by Aswinmcw — CHANGES_REQUESTED** [review]

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by VirdhatchaniKN — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]
Pls run sweeps pipeline before merging

**Review by eyonland — APPROVED** [review]

**Pull request merged**